api/graphql/schema.graphql

```diff
@@ -240,6 +243,33 @@ enum PoolStatus {
     NOT_TAKING_APPLICATIONS
 }
 
+type PoolAdvertisement {
+    id: ID!
+    expiryDate: DateTimeTz @rename(attribute: "expiry_date")
```

Comment:
probably due to 
`'expiry_date' => $this->faker->dateTimeBetween('-1 months', '1 months', 'America/Vancouver'),`
in Factory, doesn't randomly generate a time?

Reply:
It does actually have a time in the database if you check.  It's just lost somewhere.

Reply:
I think I fixed it, in Pool.php I replaced
` 'expiry_date' => 'date',`
with
` 'expiry_date' => 'datetime',`

Reply:
This is actually returning the time in UTC (+00:00) and not in the local offset - but it is correct.  This might actually easier to work with in the "time remaining" component?  I'm not sure but I'm willing to kick this topic down the road.